### ISSUE TITLE
`time_milliseconds` columns are missing in log tables

### ISSUE THREAD

**Issue by ShindongLee** [esanchezros/quickfixj-spring-boot-starter#115]
There are `time_milliseconds` columns in Quickfix MySQL event tables, but I guess they are missing in this framework. 

https://github.com/quickfix/quickfix/blob/master/src/sql/mysql/event_log_table.sql
https://github.com/quickfix/quickfix/blob/master/src/sql/mysql/messages_log_table.sql

Because they are not null columns, quickfixj-spring-boot-starter fails to store logs with their latest table definition.

Could you provide table definitions that quickfixj-spring-boot-starter is currently using? That would be very helpful.

**Comment by ShindongLee** [comment]
Could you provide table definition that quickfixj-spring-boot-starter is using? That would be very helpful.

**Comment by esanchezros** [comment]
Hi @ShindongLee, the quickfixj-spring-boot-starter doesn't define any table structure. It's a Spring Boot starter that wraps the QuickFixJ library. 

For storing events and logs in the database what Log Factory and Message Factory are you using? 

What's the error that you're getting?

**Comment by ShindongLee** [comment]
Hi @esanchezros,

I got error that `time_milliseconds` column is not null and has no default value when inserting log data to tables.
I've understood it is not the problem of `quickfixj-spring-boot-starter` 

I found [schema information](https://github.com/quickfix-j/quickfixj/blob/master/quickfixj-core/src/main/resources/config/sql/mysql/create_all.sql) in quickfixJ repository which is slightly different from the one in Quickfix repository that I mentioned.

Thank you and have a nice day!
